pkg/kubernetes/scripts/kube-client-cockpit.js

```diff
@@ -475,37 +475,27 @@
                             });
 
                             channel.addEventListener("close", function(ev, options) {
-                                var problem = options.problem || "";
                                 channel = null;
 
                                 state = 3;
-                                var cev = document.createEvent('Event');
-                                cev.initEvent('close', false, false, !!problem, 1000, problem);
+                                var cev = new Event("close", { "bubbles": false, "cancelable": false });
```

Comment:
TBH here I am bit confused as in original `initEvent` takes 6 arguments but `initEvent` is specified only with 3.

Reply:
That looks/looked like an ancient copy&paste error. The [documentation](https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent#Syntax) says `initEvent()` only takes three arguments, so the others were most probably simply ignored and we just never noticed.